aptos/token_bridge/sources/structs/transfer.move

```diff
@@ -65,19 +65,19 @@ module token_bridge::transfer {
 
     public(friend) fun create(
         amount: U256,
-        token_address: vector<u8>,
+        token_address: ExternalAddress,
         token_chain: U16,
-        to: vector<u8>,
+        to: ExternalAddress,
         to_chain: U16,
         fee: U256,
     ): Transfer {
         Transfer {
-            amount,
-            token_address,
-            token_chain,
-            to,
-            to_chain,
-            fee,
+            amount: amount,
+            token_address: token_address,
+            token_chain: token_chain,
+            to: to,
+            to_chain: to_chain,
+            fee: fee,
```

Comment:
any reason for spelling out these field names explicitly?
```suggestion
            amount,
            token_address,
            token_chain,
            to,
            to_chain,
            fee,
```
in rust, the compiler would warn about this (https://rust-lang.github.io/rfcs/1682-field-init-shorthand.html), so it's not unreasonable to expect a move compiler warning about this some time down the line.

Reply:
it was because I had `from_vector(token_address)` originally in place of token_address, so had to do `token_address:from_vector(token_address)`, and just named all the other fields for consistency. Can change this back.